man/sd_bus_send.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="sd_bus_send"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>sd_bus_send</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_send</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_send</refname>
+
+    <refpurpose>Queue a D-Bus message for transfer</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int <function>sd_bus_send</function></funcdef>
+        <paramdef>sd_bus *<parameter>bus</parameter></paramdef>
+        <paramdef>sd_bus_message *<parameter>m</parameter></paramdef>
+        <paramdef>uint64_t *<parameter>cookie</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_send()</function> queues the bus message object <parameter>m</parameter>
+    for transfer to the destination set using
+    <citerefentry><refentrytitle>sd_bus_message_set_destination</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    or its direct peer if <parameter>bus</parameter> is a direct connection. If
+    <parameter>cookie</parameter> is not <constant>NULL</constant>, it is set to the identifier
+    used to match the message with its corresponding reply. Note that in most scenarios, you don't
```

Comment:
I would split out the discussion of the destination, maybe even to a separate paragraph. After all, for majority of messages the sender just uses the default destination.

Maybe say "The parameter cookie is an output parameter. If not NULL, it is set to the identifier of the sent message."
Also, not that part in code:
```C
if (!cookie && !m->sealed)
                m->header->flags |= BUS_MESSAGE_NO_REPLY_EXPECTED;
```
I think it is worth mentioning in the page.

Reply:
I had another go at it. I don't want to add too many paragraphs as the online docs don't do any wrappings so if everything is a separate paragraph, it feels like none of them have any substance.